system-parachains/coretime/coretime-kusama/build.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
```

Comment:
not sure if this is the right license?

Reply:
Ah I hadn't seen the new licenses. I'll fix.